compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/Primitives.hs

```diff
@@ -374,65 +374,44 @@ convertPrim _ "UFetchByKey"
             , (mkIndexedField 2, EStructProj (FieldName "contract") (EVar (mkVar "res")))
             ])
 
-convertPrim version "ETemplateTypeRep"
-    ty@(TApp proxy (TCon template) :-> tTypeRep)
-    | tTypeRep `elem` [TTypeRep, TUnit] =
-    -- TODO: restrict to known templates
```

Comment:
perhaps we want to keep these TODOs?

Reply:
as you explained on slack, these TODOs were outdated anyway so better to remove them